Classes/Utility/ConfigurationUtility.php

```diff
@@ -186,10 +186,15 @@ public static function getIconPath(string $fileName): string
     public static function isValidationEnabled(array $settings, string $className): bool
     {
         $validationActivated = false;
-        foreach ((array)$settings['spamshield']['methods'] as $method) {
-            if ($method['class'] === $className && $method['_enable'] === '1') {
-                $validationActivated = true;
-                break;
+        if (\TYPO3\CMS\Core\Utility\ArrayUtility::isValidPath($settings, 'spamshield/methods')) {
```

Comment:
Can you please don't use a full qualified classname? :)

Reply:
The issue is corrected.